test/unit/interval.c

```diff
@@ -96,11 +96,62 @@ test_interval_encode_decode(void)
 	is(result.adjust, DT_EXCESS, "Adjust value is right");
 }
 
+static void
+test_interval_encode_decode_limits(void)
```

Comment:
Tests for msgpack function is good, but users will work with intervals in Lua, so I would add a test for `msgpack.encode`/`msgpack.decode` for interval objects with max numbers.

Reply:
Added pure Lua msgpack encode/decode test cases

Reply:
Thanks!